proposals/threads/Overview.md

```diff
@@ -243,6 +254,15 @@ returns the number of waiters that were woken as an `i32`.
 | `wake count` > 0 | Wake min(`wake count`, `num waiters`) waiters |
 
   * `wake`: wake up `wake count` threads waiting on the given address via `i32.wait` or `i64.wait`
+  
+For the web embedding, `i32.wait` is equivalent in behavior to executing the following:
```

Comment:
What about `i64.wait` / `i64.wake`? How do those interact with the web embedding?